README.md

```diff
@@ -5,6 +5,7 @@ A project based learning activity for people who are getting started with Git an
 ### Instructions for playing the game
 
 1. Press the space bar to begin.
```

Comment:
"Well" I know how to use the up and down, also the n and p to go back and fro:()